Framework/Core/CMakeLists.txt

```diff
@@ -15,7 +15,8 @@ set(MODULE_BUCKET_NAME O2FrameworkCore_bucket)
 O2_SETUP(NAME ${MODULE_NAME})
 if (GLFW_FOUND)
   set(GUI_SOURCES src/FrameworkGUIDebugger.cxx
-                  src/FrameworkGUIDevicesGraph.cxx)
+                  src/FrameworkGUIDevicesGraph.cxx
+                  src/PaletteHelpers.cxx)
```

Comment:
Since there is a change here which draws my attention: at some point we should move the debug GUI to a different folder than `Core`

Reply:
Yes, agreed, but I'd rather do it after I see how to do it with @dennisklein changes to the CMake stuff.